crates/nu-command/src/filters/find.rs

```diff
@@ -80,9 +80,9 @@ impl Command for Find {
                 result: None,
             },
             Example {
-                description: "Search for a term in a string",
+                description: "Search and highlight text for a term in a string",
                 example: r#"'Cargo.toml' | find toml"#,
-                result: Some(Value::test_string("Cargo.toml".to_owned())),
+                result: Some(Value::test_string("Cargo.\u{1b}[37m\u{1b}[0m\u{1b}[41;37mtoml\u{1b}[0m\u{1b}[37m\u{1b}[0m".to_owned())),
```

Comment:
I'm not sure hard-coding colors is the right approach. I'm wondering if we can use `ansi strip` here in the result. I'm not sure it's available though.

Reply:
I looked and `ansi strip` isn't available. Let's just try to get the CI green and see how to proceed from there.